### DISCUSSION TITLE
barebones starter kit

### DISCUSSION THREAD

**Discussion by ITwrx** [laravel/framework#43355] [Ideas]
I think it would be nice to have a starter kit that just set up the chosen front end stack and bundler, but without the auth, auth views, email, etc. I like to handle that stuff myself, but i don't like herding all the cats for the FE stack. 

So, an interactive script with the necessary choices for a custom, but automated, install of vue/svelte/react with inertia and tailwind or TALL, with the recommended bundler o' the day, and/or reasonable alts for bundler or css, as demand/needs dictate. 

Ready to build a modern application, but without any actual app code. 

Laravel Air? Laravel Vision? Laravel Breathe (as in "try to remember to")? :)

Thanks

**Comment by henzeb** [comment]
possibly with the ability to specify a repository on github having the configuration you want to use for all your projects? 
Something that can alter the configuration of the current project when running it again?

This could be part of the laravel installer.

**Reply by ITwrx** [reply]
i would want the ability to specify a custom remote repo, but sure, however it makes sense to address this.
